src/test/test_duplicate_registration.py

```diff
@@ -1,27 +1,31 @@
-import pytest
-
-from alembic_utils.exceptions import DuplicateRegistration
 from alembic_utils.pg_function import PGFunction
-from alembic_utils.replaceable_entity import register_entities
+from alembic_utils.replaceable_entity import register_entities, registry
 from alembic_utils.testbase import run_alembic_command
 
-TO_UPPER = PGFunction(
-    schema="public",
-    signature="to_upper(some_text text)",
-    definition="""
+
+def to_upper():
```

Comment:
it doesn't prove anything in particular in the new implementation if I'm adding literally the same object twice. This makes it so that it's comparing two instances obejcts which happen to produce the same `.identity`